puppeteer/latest.Dockerfile

```
@@ -91,22 +92,25 @@ RUN npm install -g n;
 # sbt build
+COPY puppeteer/build-puppeteer-scala-3/* /build/
 RUN sbt sbtVersion
 
 # Init yarn dependencies with latest version
-# https://classic.yarnpkg.com/lang/en/docs/cli/self-update/
-RUN npm uninstall -g yarn
-RUN touch ~/.profile
-RUN curl --compressed -o- -L https://yarnpkg.com/install.sh | bash
+# Deprecated - https://classic.yarnpkg.com/lang/en/docs/cli/self-update/
+# https://yarnpkg.com/corepack
+RUN npm install -g corepack
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)